src/components/categoricaldisplay/ResultsPanel.tsx

```diff
@@ -111,6 +112,37 @@ const CategoryTitle = ({ category, activeHeadKey, isNestedList, handleTitleClick
     </div>
 );
 
+const subCategoryMapper: {[key: string]: string} = {
+    commodity_basket: 'basket-indices/#options',
+    forex_basket: 'basket-indices/#options',
+    random_index: 'synthetic/#options',
+    crash_index: 'synthetic/#multipliers',
+    random_daily: 'synthetic/#options',
+    jump_index: 'synthetic/#options',
+    step_index: 'synthetic/#multipliers',
+    major_pairs: 'forex/#options',
+    minor_pairs: 'forex/#options',
+    americas_OTC: 'stock/#options',
+    asia_oceania_OTC: 'stock/#options',
+    europe_OTC: 'stock/#options',
+    non_stable_coin: 'cryptocurrencies/#multipliers',
+    energy: 'commodities/#options',
+    metals: 'commodities/#options',
+};
+
+const redirectLink = (subCategoryId: string) => {
+    const DEFAULT_LANGUAGE = 'EN';
+    const lang_from_url = new URLSearchParams(window.location.search).get('lang')?.toLowerCase() || DEFAULT_LANGUAGE.toLowerCase();
+
+    return `https://deriv.com/${lang_from_url}/markets/${subCategoryMapper[subCategoryId]}/`;
+};
```

Comment:
One small concern is that you'll need to consider EU users and redirect them to eu.deriv.com based on user IP

Reply:
As far as I can tell there is nothing related to countries or region that exist on smartchart repo. 

Deriv.com repo themselves uses website_status => client_country then they set region (eu/asia/america) based on hard coded list of countries that exist in their region array for redirect.

Not sure what to do lol

Reply:
I will DM you.